magefile.go

```diff
@@ -142,6 +142,10 @@ func Test() error {
 	return sh.RunV("go", "test", "./...", "-v", "2>&1", "|", "go-junit-report", ">", "junit-report.xml")
 }
 
+func TestIntegration() error {
+	return sh.RunV("go", "test", "./...", "-v", "-tags=integration")
```

Comment:
This should generate a JUnit report for the integration tests

```suggestion
	return sh.RunV("go", "test", "./...", "-v", "-tags=integration", "2>&1", "|", "go-junit-report", ">", "junit-report.xml")
```

Reply:
added it to see what happens! :-)